controller/cliController.py

```diff
@@ -7,14 +7,14 @@
 import pyfiglet
 import cmd2
 # Local Imports
-import UMLClass
-import attributes
-import relationship
-import parameter
+import model.UMLClass
```

Comment:
for each of the model classes, do _import model.<className> as <className>_

Reply:
okay